src/binder/bind/read/bind_unwind.cpp

```diff
@@ -21,7 +21,8 @@ std::unique_ptr<BoundReadingClause> Binder::bindUnwindClause(const ReadingClause
     auto aliasName = unwindClause.getAlias();
     std::shared_ptr<Expression> alias;
     if (!skipDataTypeValidation(*boundExpression)) {
-        ExpressionUtil::validateDataType(*boundExpression, LogicalTypeID::LIST);
+        ExpressionUtil::validateDataType(*boundExpression,
```

Comment:
I don't know why this will work. Technically it shouldn't because unwind operator assuming input will always be a LIST. I think you should cast to LIST if input is an array.